### PR TITLE
Add i18n

### PR DESCRIPTION
**Pull request by barmac**
This Pull Request adds i18n and `translate` module to the decision table view.

Based on https://github.com/bpmn-io/dmn-js/pull/434

### CODE REVIEW

**Comment by barmac** [comment]
There is one `TODO` in this PR, but all in all it seems to do its job. Still, I don't know why `npm run travis` fails. I have the same issue on `develop` as well as other branches.

**Comment by nikku** [comment]
Travis must have changed something with their setup. All branches, even older commits that previously passed do now fail.

File: packages/dmn-js-decision-table/src/features/decision-rules/index.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ import DecisionRules from './DecisionRules';
 
 export default {
   __depends__: [
-    ExpressionLanguagesModule
+    ExpressionLanguagesModule,
+    'translate'
```
Comment (nikku) [inline packages/dmn-js-decision-table/src/features/decision-rules/index.js]: Does this do anything? We must depend on a module. Depending on a `string` does not yield any meaningful result.

File: packages/dmn-js-decision-table/src/features/decision-rules/editor.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import DecisionRulesEditor from './DecisionRulesEditor';
 import Rules from './index';
 
 export default {
-  __depends__: [ DebounceInput, Rules ],
+  __depends__: [ DebounceInput, Rules, 'translate' ],
```
Comment (nikku) [inline packages/dmn-js-decision-table/src/features/decision-rules/editor.js]: Does this do anything? I think we must depend on a module. Depending on a `string` does not yield any meaningful result.

File: packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/InputCellContextMenu.js [hunk 3]
```diff
@@ -13,9 +13,7 @@ export default class InputCellContextMenu extends Component {
     super(props, context);
```
Comment (nikku) [inline packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/InputCellContextMenu.js]: Unneeded change. Can we remove it?

File: packages/dmn-js-decision-table/src/features/decision-table-head/editor/index.js [hunk 4]
```diff
@@ -17,7 +17,8 @@ export default {
     ContextMenu,
     DebounceInput,
     ExpressionLanguagesModule,
-    TypeRefEditing
+    TypeRefEditing,
```
Comment (nikku) [inline packages/dmn-js-decision-table/src/features/decision-table-head/editor/index.js]: Does this do anything? I think we must depend on a module. Depending on a `string` does not yield any meaningful result.

File: packages/dmn-js-decision-table/src/features/decision-table-head/index.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import DecisionTableHeadProvider from './DecisionTableHeadProvider';
 
 export default {
+  __depends__: [ 'translate' ],
```
Comment (nikku) [inline packages/dmn-js-decision-table/src/features/decision-table-head/index.js]: Does this do anything? I think we must depend on a module. Depending on a `string` does not yield any meaningful result.

**Comment by barmac** [comment]
> I'm not sure if depending on a string, i.e. `__depends__: [ 'translate' ]` does yield any meaningful result.
> 
> Getting rid of it and assuming that `translate` is provided per default is probably the better option here. We do the same in `bpmn-js`.
> 
> Other than that this one looks very good.

I'll remove that then.

**Pull request merged**